### DISCUSSION TITLE
除了自部署在docker，可否增加webdav方式的同步备份。

### DISCUSSION THREAD

**Discussion by anonymouswhac** [riskers/github-plus-extension#17] [feature]
除了自部署在docker，可否增加webdav方式的同步备份。

**Comment by riskers** [comment]
这个我不确定，还需要调研 webdav 的使用方式。

最开始本来想要支持阿里云的 faas 作为存储的，可能比 webdav 需求更大些。

后来选用 Docker，其实是想让使用者自己负责数据源的稳定性。
